test/plugin/base_test.rb

```diff
@@ -1065,14 +1126,14 @@ def setup_ec2_metadata_stubs
 
   def setup_auth_stubs
     # Used when loading credentials from a JSON file.
-    stub_request(:post, 'https://www.googleapis.com/oauth2/v3/token')
+    stub_request(:post, 'https://www.googleapis.com/oauth2/v4/token')
```

Comment:
Was this part of the logging gem update? If we use `0.24` instead, is this still needed?